src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, partnerName, isMarried },
+}) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">
+          <span>{`I am ${age}`}</span>
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried === true && sex === 'm'
```

Comment:
```suggestion
        {isMarried && sex === MAN
```
Consider using constants with meaningful names instead of magic strings